helm/patroni/README.md

```diff
@@ -0,0 +1,147 @@
+# Patroni Helm Chart (forked from helm chart incubator)
```

Comment:
helm chart incubator is EOL; let's check with Zalando on where they intend to maintain this moving forward?